corehq/apps/userreports/pillow.py

```diff
@@ -250,7 +260,7 @@ def _save_doc_to_table(self, domain, table, doc, eval_context):
             table.best_effort_save(doc, eval_context)
         except UserReportsWarning:
             # remove it until the next bootstrap call
-            self.table_manager.table_adapters_by_domain[domain].remove(table)
+            self.table_manager.remove_adapter(domain, table)
```

Comment:
is this removing an adapter or a table? that is a bit confusing here.

Reply:
an adapter